core/src/test/java/bio/terra/pearl/core/service/export/EnrolleeImportServiceTests.java

```diff
@@ -406,6 +473,32 @@ private Import doImport(StudyEnvironmentFactory.StudyEnvironmentBundle bundle, S
                 admin.getId(), fileType);
     }
 
+    private void verifySurvey(ImportItem importItem, String surveyStableId, String questionStableId, String questionAnswer) {
+        List<ParticipantTask> tasks = participantTaskService.findByEnrolleeId(importItem.getCreatedEnrolleeId());
+        assertThat(tasks, hasSize(1));
+        List<Answer> answers = answerService.findByEnrolleeAndSurvey(importItem.getCreatedEnrolleeId(), surveyStableId);
+        assertThat(answers, hasSize(1));
+        Answer diagnosis = answers.stream().filter(answer -> answer.getQuestionStableId().equals(questionStableId))
+                .findFirst().get();
+        assertThat(diagnosis.getStringValue(), equalTo(questionAnswer));
+    }
+
+    private void verifyKitRequest(Import dataImport, List<KitRequestDto> expectedKitRequests) {
```

Comment:
this method should take an ImportItem, or some other way of indicating which enrollee should have kit requests.  As is, you're not confirming the kits got created for the right person.